docs/deployment_configuration.md

```diff
@@ -15,7 +15,7 @@
 - "BATCH_SIZE" - Datadog batch size for APM
 - "SYNC_THRESHOLD" - Datadog sync threshold for APM
 - "ETHEREUM_EVENTS_CHECK_INTERVAL_MS" - the frequency of HTTP requests towards the Ethereum clients and scanning for interested events. Should be less then average block time (10 to 20 seconds) on Ethereum mainnet.
-
+- "ETHEREUM_STALLED_SYNC_THRESHOLD_MS" - the threshold before considering an unchanging Ethereum block height to be considered a stalled sync. Should be slightly larger than the expected block time.
```

Comment:
> Should be slightly larger than the expected block time.

"slightly larger" i.e. 20s over 15s, sounds very aggressive. It is OK to be like this in the `alarm.get` endpoint (more detailed and sensitive), but for `status.get` byz events it might be too aggressive. byz events are also informally called "actionable items" so whenever there's something there, the user is expected to take some action.

Not sure how to accommodate this though (and if at all). Just throwing a thought in the air.

Reply:
I guess let's try with this value. More aggressive now means we can see its behaviour earlier and more often, then we can readjust the threshold based on what we see. It's configurable anyway. 🎉

Reply:
ah, if this moves to `alarm.get` then aggressive is totally fine IMHO

Reply:
Aha I got it. Your main concern is polluting byz events. Lets have these alarms in `/alert.get` only then.